docs/whirlpool/docs/03-Whirlpools SDK/03-Position Management/01-Open Position.md

```diff
@@ -0,0 +1,92 @@
+- **RPC Setup**: Use a Solana RPC client to communicate with the blockchain.
+- **Wallet Creation**: Create a wallet to interact with the Solana network.
+- **Devnet Airdrop**: Fund your wallet with a Solana Devnet airdrop to cover transaction fees.
+
+For more details, refer to our [Environment Setup Guide](../01-Environment%20Setup.md)
+
+### Opening a Position in Splash Pools
+
+1. **Pool Address**: Provide the address of the Splash Pool where you want to open a position.
+2. **Liquidity Parameters**: Choose how you want to provide liquidity. You only need to provide one of these parameters, and the function will compute the others in the returned quote based on the current price of the pool:
+    - `liquidity`: Specify the liquidity value to provide.
+    - `tokenA`: Specify the amount of token A (first token in the pool).
+    - `tokenB`: Specify the amount of token B (second token in the pool).
+3. **Slippage Tolerance**: Set the maximum slippage tolerance (optional, defaults to 1%). Slippage refers to the difference between the expected price and the actual price at which the transaction is executed. A lower slippage tolerance reduces the risk of price changes during the transaction but may lead to failed transactions if the market moves too quickly.
+4. **Funder**: This will be your wallet, which will fund the transaction.
+5. **Create Instructions**: Use the `openFullRangePositionInstructions()` function to generate the necessary instructions.
+    ```tsx
+    const { quote, instructions, initializationCost } = await openFullRangePositionInstructions(
+        devnetRpc,
+        poolAddress,
+        param, 
+        slippageTolerance,
+        wallet
+    );
+    ```
+6. **Submit Transaction**: Include the generated instructions in a Solana transaction and send it to the network using the Solana SDK. Ensure that you have enough of both Token A and Token B as calculated in the quote, or the transaction will fail.
+
+### Opening a Position in Concentrated Liquidity Pools
+
+1. **Pool Address**: Provide the address of the Splash Pool where you want to open a position.
+2. **Liquidity Parameters**: Choose how you want to provide liquidity. You only need to provide one of these parameters, and the function will compute the others in the returned quote based on the price range and the current price of the pool:
+    - `liquidity`: Specify the liquidity value to provide.
+    - `tokenA`: Specify the amount of token A (first token in the pool).
+    - `tokenB`: Specify the amount of token B (second token in the pool).
+3. **Price Range**: Set the lower and upper bounds of the price range within which your liquidity will be active. The current price and the specified price range will influence the quote amounts. If the current price is in the middle of your price range, the ratio of token A to token B will reflect that price. However, if the current price is outside your range, you will only deposit one token, resulting in one-sided liquidity.
```

Comment:
Maybe add a note that positions only earn fees if the price falls within you selected price range?